src/components/code-snippet/code-snippet.js

```diff
@@ -41,7 +41,6 @@ class CodeSnippet extends mixin(createComponent, initComponentBySearch, handles)
 
     if (this.element.offsetHeight < this.options.minHeight) {
       this.element.classList.add(this.options.classHideExpand);
-      this.element.classList.add(this.options.classExpanded);
```

Comment:
@asudoh This change was intentional. When I added the styling for the experimental component, the multi line code snippet would initiate already expanded, but the button would still read `Show More`. Then when I clicked it, it would fix to `Show Less`, and it would work perfectly fine after that. When I disabled the feature-flag to check if the standard multi line code snippet had that same issue, it didn't, it would initiate normally. When I removed that line of code, the experimental component would initiate properly and so would standard. I'm not sure what stable mode is.